vaadin-flow-components-shared-parent/vaadin-flow-components-base/src/main/resources/META-INF/resources/frontend/tooltip.ts

```diff
@@ -0,0 +1,10 @@
+import { Tooltip } from '@vaadin/tooltip';
+
+const Vaadin = (window as any).Vaadin ||= {};
+const Flow = Vaadin.Flow ||= {};
```

Comment:
Have you tried if this new syntax `||=` works with webpack? 
IIRC it should, as we have compile target for TypeScript set to `es2019`.

Reply:
The IT page at least runs nicely with the feature flag on.